Classes/Services/MetsExporter.php

```diff
@@ -793,6 +795,16 @@ public function getSlubData()
     }
 
     /**
+     * replaces single quote with double quote
+     * @param $string
+     * @return string
+     */
+    function replaceSingleQuote($string)
```

Comment:
I did not talk about source code, but about binary code which is smaller and faster for static functions because those functions don't need the object information (which is passed as hidden argument for non-static class functions). It's a different question whether saving a few bytes and a few microseconds are important. IMHO it's more important that the keyword documents that a function does not access object information.